src/batch-client/index.ts

```diff
@@ -489,6 +489,18 @@ export class ZimbraBatchClient {
 			}
 		}).then(res => normalize(FreeBusy)(res.usr));
 
+	public generateScratchCodes = (username: String) =>
+		this.jsonRequest({
+			name: 'GenerateScratchCodes',
+			namespace: Namespace.Account,
+			body: {
```

Comment:
Same as https://github.com/Zimbra/zm-api-js-client/pull/205/files#r292307415

Reply:
username is mandatory parameter as per soap api